cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -48,17 +48,19 @@ pub enum SnowbridgeControl {
 	Control(ControlCall),
 }
 
+/// Create an agent on Ethereum. An agent is a representation of an entity in the Polkadot
+/// ecosystem (like a parachain) on Ethereum.
```

Comment:
I don't think rust-doc `///` comments are meaningful for tests. Should probably just use `//`

```suggestion
// Create an agent on Ethereum. An agent is a remote proxy for a consensus system within Polkadot 
```

Reply:
I thought the same but then I checked what Parity does, and it looks like they use `///` for test comments fairly consistently:

- https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/client/consensus/common/src/tests.rs#L1274
- https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/pallets/dmp-queue/src/tests.rs#L153
- https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/pallets/xcmp-queue/src/tests.rs#L117

Reply:
ah ok, :+1: